assets/payload.json

```diff
@@ -0,0 +1,50 @@
+{
+    "plugin": "kubejobs",
+    "plugin_info": {
+        "username": "admin",
+        "password": "grafana123123",
+        "cmd": [
+            "python",
+            "application.py"
+        ],
+        "img": "matheusmelo/sort_strings_no_swift",
+        "init_size": 1,
+        "redis_workload": "https://gist.githubusercontent.com/GabrielSVinha/cf2ca2604160aedbc7675aac60cccc77/raw/47b655333742d4fe0a403b6fba48137527034d62/workload",
```

Comment:
Same as above, we could use a shorter workload so the cluster won't be overloaded

Reply:
+1 for a shorter workload. what's the estimated time for this amount of items? (just to make sure `expected_time` and `max_replicas` also make sense here).

Reply:
@clenimar I believe it's around 2mins. We should discuss this whole json file to make it adequate for this use case.

Reply:
Done.